spec/models/game_spec.rb

```diff
@@ -43,4 +43,47 @@
       expect(game.white_to_move).to eq(true)
     end
   end
+
+  describe "Populate Board" do
+    it "should have correct number of pieces" do
+      expect(game.pieces.count).to eq 32
+    end
+
+    it "should successfully hook up the database relationship between pieces and games" do
+      piece = game.pieces.last
+      expect(piece.game).to eq(game)
+    end
+
+    it "should be empty on the squares" do
+      expect(game.pieces.where(rank: 3..6)).to eq([])
+    end
+
+    (1..8).each do |file|
+      it "should have a white pawn at rank 2, file #{file}" do
+        @white_pawn = game.pieces.find{|piece| piece.rank == 2 && piece.file == file}
```

Comment:
An instance variable declared in an `it` block will actually be available across the entire `describe` block that the specific example belongs to and can cause tests to interfere with each other. Just use local variables instead.